### DISCUSSION TITLE
Reset password flow

### DISCUSSION THREAD

**Discussion by chiiya** [filamentphp/filament#1309] [Ideas]
For v2 the reset password flow has been removed. Is there a reason for this? Would you be open to a merge request that adds this functionality again? I'm thinking of a `passwords` config option, similar to `guard`, to configure the password broker that should be used.

**Comment by danharrin** [comment]
Someone is currently building password resets as a plugin, I think. In v2 we decided to focus on maintaining the unique parts of Filament, and letting others choose what auth system they want instead of us spending time on it. We used to commonly get requests to customise the entire auth flow, including the password resets. Now, you can just use your own `Login` Livewire component in `config/filament.php`, which allows you to have a completely custom authentication & password reset behaviour.

**Comment by danharrin** [comment]
Check out https://github.com/jeffgreco13/filament-breezy!

**Reply by grahammtbr** [reply]
> Check out https://github.com/jeffgreco13/filament-breezy!

Well that's cool! Saves me the time later this week when I was going to try and mash Breeze and Filament together.
